api/client/catalog/categories.go

```diff
@@ -9,17 +9,20 @@ import (
 )
 
 type CategoryCreate struct {
-	Name string `json:"name"`
+	Name      string `json:"name"`
+	ProjectId string `json:"projectId"`
```

Comment:
How is `ProjectId` generated? Is it specified in the YAML by the user, or is it generated by the CLI/API?

Reply:
specified by the user @shekhar-rudder